app/MMB/src/menuGroups/PassengerMenuGroup.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import * as React from 'react';
+import { TextIcon } from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+
+import TitledMenuGroup from '../components/menu/TitledMenuGroup';
+import MenuItem from '../components/menu/MenuItem';
+
+type Props = {|
+  openSubmenu: (activeId: string, menuId: string) => void,
+  activeId: string,
+|};
+
+const VoidAction = () => {
+  console.warn('TODO');
+};
+
+export default class PassengerMenuGroup extends React.Component<Props> {
+  render = () => (
+    <TitledMenuGroup title={<Translation id="mmb.main_menu.passengers" />}>
+      <MenuItem
+        onPress={VoidAction} // TODO: Add handler
+        title={<Translation id="mmb.main_menu.details" />}
+        isActive={false} // TODO: Add logic
+        icon={<TextIcon code="&#xe087;" />}
+      />
```

Comment:
TODO: bags (? not sure about that), passengers details + visa info

![screen shot 2018-05-15 at 15 15 46](https://user-images.githubusercontent.com/978368/40058917-f01a71c6-5852-11e8-8c8d-e40c905a3c32.png)

Reply:
Maybe bags shows up there if it is ordered?

Reply:
That sounds reasonable but you can find the bags in the passenger as well. Not sure about the flow here... 🤷‍♂️

Reply:
Maybe some bags contains passengers? 🙈